src/databrowser.js

```diff
@@ -165,7 +165,7 @@ define([
                 }
                 info = info._parent;
             }
-            vellum.handleDropFinish(target, path);
+            vellum.handleDropFinish(target, path, widget.mug);
```

Comment:
Maybe I should have passed in `null` as the third argument to be more explicit. It was intentional that I did not supply a third argument in this call.